app/views/admin/users/index/_controls.html.erb

```diff
@@ -1,16 +1,30 @@
-<% if params[:controller] == "admin/users" %>
-  <%= form_with url: admin_users_path, method: :get, local: true, class: "flex flex-col m:flex-row gap-3 m:items-center p-3" do |f| %>
-    <div class="crayons-field flex-1 flex-row items-center gap-2">
-      <%= f.label :search, class: "crayons-field__label whitespace-nowrap" %>
-      <%= f.text_field :search, value: params[:search], class: "crayons-textfield mt-0", placeholder: "Name, username, email, or Twitter/GitHub usernames", aria: { label: "Filter by name, username, email, or Twitter/GitHub usernames" } %>
-    </div>
-    <div class="crayons-field flex-row items-center gap-2">
-      <%= f.label :role, "Filter by Role", class: "crayons-field__label whitespace-nowrap" %>
-      <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: true }, class: "crayons-select mt-0" %>
-    </div>
-    <%= f.submit "Filter", class: "c-btn c-btn--secondary" %>
+<% if FeatureFlag.enabled?(:member_index_view) %>
```

Comment:
I decided to tackle the search and filter fields while I was in this file since I was already tinkering with the layout. Since these changes are incredibly minimal, I decided the keep the updated fields within the existing partial, rather than pulling them out for the time being.

Reply:
Awesome! I'm going to pick up the filters/pagination issue next, and I'll build off of this starting point ✨